src/cpp/include/openvino/genai/scheduler_config.hpp

```diff
@@ -16,7 +16,7 @@ struct SchedulerConfig {
     std::size_t num_kv_blocks = 0;
 
     // total size of KV cache in GB
-    std::size_t cache_size = 0;
+    std::size_t cache_size = 1;
```

Comment:
why? it's explicitly set to 0, so users have to set some meaningful values on their own

Reply:
It's now possible to construct the struct with all the default values which is used to create CB backend in LLMPipeline.

Reply:
ok, let's fix it together with CB device name via passing scheduler config as device property

Reply:
@olpipi, please, address it